thirdeye-ui/src/app/components/welcome-landing-page/welcome-step-card/welcome-step-card.component.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2022 StarTree Inc
+ *
+ * Licensed under the StarTree Community License (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at http://www.startree.ai/legal/startree-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT * WARRANTIES OF ANY KIND,
+ * either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { Box, Button, Typography } from "@material-ui/core";
+import CheckCircleIcon from "@material-ui/icons/CheckCircle";
+import React, { FunctionComponent } from "react";
+import { DimensionV1 } from "../../../platform/utils";
+import type { WelcomeStepCardProps } from "./welcome-step-card.interfaces";
+import { useWelcomeStepCardStyles } from "./welcome-step.card.styles";
+
+export const WelcomeStepCard: FunctionComponent<WelcomeStepCardProps> = ({
+    title,
+    subtitle,
+    ctaContent,
+    disabled = false,
+    isComplete = false,
+    onClickCta = () => null,
```

Comment:
We should agree on whether doing `onClickCta && onClickCta()` in the places that use a callback function or providing a noop

Reply:
ok then we will need to change all the `callbackFunctionProperty && callbackFunctionProperty()`  to noop at the top and then `callbackFunctionProperty()` when encountered going forward. Or do a one time then and refactor

Reply:
Wait on second thought, I just encountered a situation where leaving a callback function empty or falsey causes different styling in a component. Let keep the original pattern as that leaves that ability to determine different styling

Reply:
So I'll revert this change to have a default callback function if none is passed, if I understand correctly.

Reply:
Also, if needed, we can use optional chaining again (`callback?.()`) instead of `callback && callback()`



https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
Reverted to using a default callback

Reply:
Good call on `callback?.()`, will do that going forward